charts/clearblade-monitoring/charts/cb-monitoring/templates/monitoring-config.yaml

```diff
@@ -49,10 +49,70 @@ data:
        annotations:
          summary: "Pod {{ "{{" }} $labels.pod }} is dropping messages"
       {{- end}}
+      {{- if $.Values.alerts.adminConnections }}
+     - alert: HighDatabaseConnections
+       expr: pg_stat_database_numbackends{datname="admin",namespace="{{ $n.name }}"} > {{ $.Values.alerts.maxAdminConnections }}
+       for: 5m
+       labels:
+         severity: warning
+       annotations:
+         summary: "High number of database connections to admin database"
+         description: "The number of connections to the admin database has exceeded the threshold of {{ $.Values.alerts.maxAdminConnections }}. "
+      {{- end}}
+      {{- if $.Values.alerts.HTTPFailure }}
+     - alert: HighHttpFailureRate
+       expr: rate(clearblade_http_failed{namespace="{{ $n.name }}"}[5m]) > {{ $.Values.alerts.HTTPFailureValue }}
+       for: 5m
+       labels:
+         severity: warning
+       annotations:
+         summary: "High HTTP failure rate"
+         description: "The HTTP failure rate is above {{ $.Values.alerts.HTTPFailureValue }}/second for past 5 minutes"
+      {{- end}}
+      {{- if $.Values.alerts.HTTPFailureDeriv }}
+     - alert: IncreasingHttpFailures
+       expr: deriv(clearblade_http_failed{namespace="{{ $n.name }}"}[10m]) > {{ $.Values.alerts.HTTPFailureDerivValue }}
+       for: 5m
+       labels:
+         severity: warning
+       annotations:
+         summary: "Rapidly increasing HTTP failures"
+         description: "The number of HTTP failures is increasing rapidly"
+      {{- end}}
+      {{- if $.Values.alerts.lowMQTTConnections }}
+     - alert: LowMqttConnections
+       expr: sum(clearblade_mqtt_current_connections{namespace="{{ $n.name }}"}) < {{ $.Values.alerts.lowMQTTConnectionsValue }}
+       for: 5m
+       labels:
+         severity: warning
+       annotations:
+         summary: "Low MQTT connections"
+         description: "The number of MQTT connections is below 10"
+      {{- end}}
+      {{- if $.Values.alerts.MQTTDisconnectDeriv }}
+     - alert: RapidMqttConnectionLoss
+       expr: sum(-deriv(clearblade_mqtt_current_connections{namespace="{{ $n.name }}"}[10m])) > {{ $.Values.alerts.MQTTDisconnectDerivValue }}
+       for: 5m
+       labels:
+         severity: warning
+       annotations:
+         summary: "Rapid loss of MQTT connections"
+         description: "The number of MQTT connections is dropping rapidly."
+      {{- end }}
+      {{- if $.Values.alerts.license }}
+     - alert: LicenseExpiring
+       expr: clearblade_expiration{instance="clearblade-monitoring-service-0.{{ $n.name }}.svc.cluster.local"} < {{ $.Values.alerts.licenseValue }}
+       for: 5m
+       labels:
+         severity: warning
+       annotations:
+         summary: "License due to expire soon"
+         description: "License expires in {{ $.Values.alerts.licenseValue }} days!"
```

Comment:
Doesn't this alert mean that the license expires in **less than** this number of days? Can we report the actual number of days until expiration?